safekeeper/src/bin/safekeeper.rs

```diff
@@ -120,7 +122,8 @@ struct Args {
     log_format: String,
 }
 
-fn main() -> anyhow::Result<()> {
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Why do you need only single-threaded env?

Reply:
Because this is for the main task, which does almost nothing.